Sources/NIOHTTP1/HTTPTypes.swift

```diff
@@ -335,6 +335,33 @@ public struct HTTPHeaders: CustomStringConvertible, ExpressibleByDictionaryLiter
         }
     }
 
+    /// Add a sequence of header name/value pairs to the block.
+    ///
+    /// This method is strictly additive: if there are other entries with the same header
+    /// name already in the block, this will add new entries. `add` performs case-insensitive
```

Comment:
This comment looks wrong: we don't compare the field names at all, except inasmuch as we're looking for the Connection header.